src/ts/components/subtitleoverlay.ts

```diff
@@ -326,6 +331,10 @@ class ActiveSubtitleManager {
       text: event.html || ActiveSubtitleManager.generateImageTagText(event.image) || event.text,
     });
 
+    if (event.regionStyle) {
+      label.getDomElement().attr('style', event.regionStyle + ';position:absolute');
```

Comment:
Are you sure that the right way to handle TTML cues is to set the region style on the label itself (in contrast to e.g. putting the labels into a container with the region style)? I'm thinking about multiple labels at the same time in the same region here... with this implementation they would overlap.